### PR TITLE
Updated invest/expand home page

### PR DESCRIPTION
**Pull request by jessgilbert**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] (if changing the UI) Add a printscreen to the PR

image
image
image
image
image
image
image
### Order of sections were from Bo's design, but we were not 100% on the ordering so have not rushed to release

### CODE REVIEW

File: invest/templates/invest/landing_page.html [hunk 1]
```diff
@@ -97,81 +97,60 @@ <h2 class="heading-large">{{ page.eu_exit_section_title }}</h2>
...
 {% endif %}
 
-{% if page.sector_title and page.sectors|filter_by_active_language %}
-<section class="landing-page-section" id="industries">
-    <div class="container">
-        <h2 class="heading-large">{{ page.sector_title }}</h2>
-            <div class="grid-row padding-bottom-45">
-                <p class="font-small column-half"> {{  page.sector_intro }} </p>
+{% if page.how_to_expand_title and page.how_to_expand_image.url and how_to_expand_items_length > 0 %}
+<section class="how-to-expand" style="background-image: url('{{ page.how_to_expand_image.url }}')">
```
Comment (MichaelWalker) [inline invest/templates/invest/landing_page.html]: I think there is already a component for this - can we use that instead?

image

Comment (jessgilbert) [reply invest/templates/invest/landing_page.html]: Oh yeah! This was my branch before I added that to directory components forgot to then use it

File: core/sass/pages/invest/_landing-page.scss [hunk 2]
```diff
@@ -26,6 +26,22 @@
...
+    color: white;
+    a {
+      color: white;
+    }
+
+    .how_to_expand_item {
+      border-top: 3px solid $great-red;
+    }
+  }
+
```
Comment (MichaelWalker) [inline core/sass/pages/invest/_landing-page.scss]: do we still need this? Looks like it might now be covered by directory-components?

Comment (jessgilbert) [reply core/sass/pages/invest/_landing-page.scss]: Yep woops, removed now!

**Comment by jessgilbert** [comment]
Design not entirely signed off (ordering of sections was debated) have updated the ticket to reflect this

**Pull request closed without merging**